src/components/common/GProfilerModal.tsx

```diff
@@ -0,0 +1,335 @@
+import _ from 'lodash';
+import { BackendAPI } from './API';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+
+import React, { Component } from 'react';
+import {
+    Button,
+    Modal,
+    Form,
+    Checkbox,
+    Table,
+    Input,
+    Select,
+    Label,
+} from 'semantic-ui-react';
+import { instanceOf } from 'prop-types';
+import { withCookies, Cookies } from 'react-cookie';
```

Comment:
You're not actually using cookies here are you? All the cookie code could be removed if not.

Reply:
Yeah that's right, will remove